magento/pwa-studio
# Fix dev server unhandled error

## Description
Fix an unhandled error wherein the dev server attempts to read tap into the webpack compilation to read GraphQL query files. It looks like the shape of an argument changed somewhere.

## Related Issue
Maybe related to PWA-169.

## Acceptance 

### Verification Stakeholders

### Specification

### Verification Steps


1. `yarn watch:venia`
2. Verify that the node process does not warn about an unhandled error.
3. Verify that the app is successfully built and served.

## Screenshots / Screen Captures (if appropriate)

## Checklist
* I have updated the documentation accordingly, if necessary.
[ ] I have added tests to cover my changes, if necessary.
